data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,198 @@ mission "Remnant: Visit to Paradise 1"
+			`	"Well met, <first>. We were just going to the monthly Celebration of Remembrance. Would you like to accompany us?"`
+			choice
+				`	"Yes, I would."`
+					goto celebrationattendance
+				`	"What is that?"`
+					goto celebrationdefinition
+				`	"No thanks."`
+			label celebrationdeclined
+			`	She nods. "May the Embers guide you, <first>." With that, she rejoins the stream of people and vanishes from sight.`
+				decline
+			label celebrationdefinition
+			`	"The Celebration of Remembrance is a gathering where we take a moment to remember where we have come from, appreciate where we are, and express our hope for the future." Her gestures take on a critical tone. "That sounds very formal, but it is more artistic than defined." She glances at the slowing flow of people. "It will be starting shortly, and I want to attend this month, so I should get moving. Do you want to come too?"`
+			choice
+				`	"Yes, I do."`
+				`	"No thanks."`
+					goto celebrationdeclined
+			label celebrationattendance
+			`	As the two of you join the flow of people Taely gestures once again. "I can provide you with some commentary if you like, but it is more about feeling than doing anything specific. Just to provide you with some context, this is a free-form dance mixed with singing and music. You are welcome to participate. Just try not to be disruptive, such as getting in the way of people dancing."`
+			choice
+				`	"No, that is alright. I will just experience it."`
+				`	"Yes, that would be appreciated."`
+					goto celebrationannotated
+			`	The flow of people you are a part of joins with other streams of people, finally leading out of the typical hallways and into a large cave. A large opening on one side opens upwards towards the night sky, framed by rough stone and a handful of shield emitters. Stalagmites ring the perimeter of the room with matching stalactites above them, but there is no trace of drips or moisture here. While the walls are rough natural stone, the floor and stalagmites have been polished smooth. At a quick glance, you estimate that there are probably over a hundred people in here with more filtering in through several doors. Taely leads you through the crowd, frequently gesturing greetings to those she passes by.`
+			`	Without warning the room stills, and the doors close. You see a flurry of gestures wash over the room, as if everyone is repeating what someone closer to the center is saying, but you cannot see anyone starting it. "Perimeter secure?" it asks, followed by a ripple of gestures from the edges of the room back toward the center: "All clear."`
+			`	Suddenly you are aware of a drum beat building in the silence, and realize that it had started with the first ripple of gestures, so quiet that it was unnoticeable. The beat builds gradually, a slow, steady rhythm.`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
+			`	"Thump."`
+			choice
+				`	(Look around the room.)`
+				`	(Move in time with the beat.)`
```

Comment:
Correct. This is both a break in the flow and an opportunity to let the player do something. Neither option affects the story, though. Which is logical, since the player isn't the focus of what is going on, and has no bearing on the events. As such, this is more an RP detail so that players can decide for themselves if they're just observing or actually taking part.